src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -182,36 +173,14 @@ public void Should_not_add_everyone_and_anonymous_to_already_existing_queues()
             Assert.IsNull(accessControlEntryType);
         }
 
-        [Test]
-        public void Should_allow_queue_names_above_the_limit_for_set_permission()
```

Comment:
Since this test has been removed, does this mean that some functionality in the transport itself should change to make this scenario invalid?